src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -91,6 +95,16 @@ export const ConversationSpace: React.FunctionComponent = () => {
               )}
             </div>
           </div>
+          {!seatedAtTable && (
+            <TablesControlBar
+              className="control-bar"
+              defaultTables={defaultTables}
+              venue={venue}
+              users={recentVenueUsers}
+              onToggleTables={setTables}
+              isChecked={tables !== defaultTables}
```

Comment:
hmmm, not sure if we should use that approach to understand if the input was toggled. Can we use `useShowHide` hook in the component instead and rewrite the logic?
```
tables !== defaultTables
```

Reply:
fixed in 5aa6329bd90551d39634b475d84adbb35e5e663a